extra_geom/__init__.py

```diff
@@ -35,10 +35,10 @@
 
 __version__ = "1.3.0"
 
-from .detectors import (
-    AGIPD_1MGeometry, AGIPD_500K2GGeometry, GenericGeometry, DSSC_1MGeometry, JUNGFRAUGeometry,
-    LPD_1MGeometry, PNCCDGeometry, Epix100Geometry, Epix10KGeometry, agipd_asic_seams,
-)
+from .detectors import (AGIPD_1MGeometry, AGIPD_500K2GGeometry,
+                        DSSC_1MGeometry, Epix10KGeometry, Epix100Geometry,
+                        GenericGeometry, JUNGFRAUGeometry, LPD_1MGeometry,
+                        PNCCDGeometry, agipd_asic_seams)
```

Comment:
I prefer the previous style (with names on 2 lines isntead of 4) on this one.

Reply:
Yep, no problem, done